dockerfiles/plain_go_enclave_local.Dockerfile

```diff
@@ -1,9 +1,18 @@
 FROM golang:1.17-alpine
 
+# on the container:
+#   /data                    contains working files for the enclave
+#   /home/obscuro/go-obscuro contains the src
+#
+# Note: on ego it maps /home/obscuro/data to /data inside the enclave,
```

Comment:
This is the mapping of the virtual fs yeah ?
Could that be (more) explicit  ?

Reply:
Yeah exactly, I've updated to try and make the wording more clear, it's a bit of a fudge tbh, maybe we make the enclave data dir configurable eventually.